chore: deployed the `create fuels` template (#3045)
FuelLabs/fuels-ts: apps/docs/src/guide/creating-a-fuel-dapp/index.md

![End result of this guide](../../public/creating-a-fuel-dapp-create-fuels-end-result.png)

> Or you can test out a version of the create fuels dApp deployed to testnet [here](https://create-fuels-template.vercel.app/).

Reviewer: Nit.

```suggestion
You can also check it live, deployed to the Testnet:
 - https://create-fuels-template.vercel.app/
```
